src/components/Dropdown/Dropdown.css

```diff
@@ -67,8 +67,15 @@
 	border-bottom: 1px solid var(--color--gray--light);
 }
 
+.dropdown-menu > *:first-child {
+	border-top-left-radius: var(--dropdown--menu--border-radius);
```

Comment:
I did not know that this was a thing you could set with css, very cool!

Reply:
Do you mean CSS variables, or setting the border radius?